common/rpc.go

```diff
@@ -24,6 +24,23 @@ import (
 	"go.uber.org/yarpc"
 )
 
+const (
+	// LibraryVersionHeaderName refers to the name of the
+	// tchannel / http header that contains the client
+	// library version
+	LibraryVersionHeaderName = "cadence-client-library-version"
+
+	// FeatureVersionHeaderName refers to the name of the
+	// tchannel / http header that contains the client
+	// feature version
+	FeatureVersionHeaderName = "cadence-client-feature-version"
```

Comment:
client change has already merged: https://github.com/uber-go/cadence-client/pull/316

Reply:
What I meant is we need to document this feature version on server.

Reply:
added:

// the feature version sent from client represents the
// feature set of the cadence client library support.
// This can be used for client capibility check, on
// Cadence server, for backward compatibility

Reply:
// the feature version sent from client represents the
// version of the server feature set expected by the client.
// Cadence service is expected to reject clients that have incompatible major version.
// In some cases a client with a smaller major version than the service is allowed for the backwards compatibility. Clients with a larger major version are always rejected.